### PR TITLE
Location endpoint

### PR DESCRIPTION
**Pull request by jmrr**
Further improvements:

- No string interpolation in `conn.execute()`
- Use `schema` for validation
...

### CODE REVIEW

File: api/handler.py [hunk 1]
```diff
@@ -13,25 +14,22 @@ def make_conn():
...
 def postLocation(event, context):
-    # body = {
-    #     "message": "POST /location endpoint logic. Your function executed successfully!",
-    #     "input": event,
-    # }
-
-    # response = {"statusCode": 200, "body": json.dumps(body)}
+    payload = json.loads(event["body"])
+    point_str = f"'POINT({payload['longitude']} {payload['latitude']})'"
+    sql_statement = f"INSERT INTO contact_tracer.device_location (device_id,sample_date,location) VALUES ('{payload['deviceId']}', '{payload['sampleDate']}', postgis.ST_PointFromText({point_str}))"
```
Comment (cyterdan) [inline api/handler.py]: think you forgot the upsert into the device table, that's needed to have a list of devices to iterate over for score calculation

File: api/handler.py [hunk 2]
```diff
@@ -13,25 +14,22 @@ def make_conn():
...
 
 def postLocation(event, context):
-    # body = {
-    #     "message": "POST /location endpoint logic. Your function executed successfully!",
-    #     "input": event,
-    # }
-
-    # response = {"statusCode": 200, "body": json.dumps(body)}
+    payload = json.loads(event["body"])
+    point_str = f"'POINT({payload['longitude']} {payload['latitude']})'"
```
Comment (cyterdan) [inline api/handler.py]: I really like the previous version where you included the SRID as well

**Review by cyterdan — CHANGES_REQUESTED** [review]

File: api/handler.py (hunk 2 above)
Comment (jmrr) [reply api/handler.py]: yeah will add it, just wanted to get a working version asap.

File: api/handler.py (hunk 1 above)
Comment (jmrr) [reply api/handler.py]: Done

File: api/handler.py (hunk 2 above)
Comment (jmrr) [reply api/handler.py]: Done.

**Pull request merged**